ai-data/generative-apis/api-cli/using-chat-api.mdx

```diff
@@ -0,0 +1,88 @@
+---
+meta:
+  title: Using Chat API
+  description: This page explains how to use the Chat API to query models
+content:
+  h1: Using Chat API
+  paragraph: This page explains how to use the Chat API to query models
+tags: generative-apis ai-data chat-api
+dates:
+  validation: 2024-09-03
+  posted: 2024-09-03
+---
+
+Scaleway Generative APIs are designed as a drop-in replacement for the OpenAI APIs. If you have an LLM-driven application that uses one of OpenAI's client libraries, you can easily configure it to point to Scaleway Chat API, and get your existing applications running using open-weight instruct models hosted at Scaleway.
+
+## Create chat completion
+
+Creates a model response for the given chat conversation.
```

Comment:
```suggestion
Run the following command to create a model response for the given chat conversation:
```

Reply:
Not sure. Usually each API method is introduced by a short descriptive sentence about what it does. It wasn't meant to be an action for the user. I can add another line for this purpose if you think we need one before the CURL that follows.